care/facility/api/serializers/patient.py

```
@@ -47,6 +47,37 @@
 from care.utils.serializers.fields import ChoiceField, ExternalIdSerializerField
 
 
+class PatientRegistrationSerializer(serializers.ModelSerializer):
+    member_id = serializers.CharField(required=True)
+    policy_id = serializers.CharField(required=True)
+    insurer_id = serializers.CharField(required=True)
+    insurer_name = serializers.CharField(required=True)
+
+    class Meta:
+        model = PatientRegistration
+        fields = '__all__'
```

Ruff

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)